src/views/Dashboards/Dashboards.js

```diff
@@ -5,8 +5,8 @@ import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 import { Card, CardContent, Grid, Tab, Tabs } from '@mui/material';
 import { useRedirectFromDisabledView } from '../../hooks/RouterHooks';
-import { useDashboardsViewReportsConfig } from '../../state/powerBi/hooks';
-import { DashboardsPowerBiReport } from './components';
+import { useDashboardsViewReportsConfig } from '../../state/charts/hooks';
+import { DashboardsChartReport } from './components';
```

Comment:
Since the component is already in the Dashboards view folder, could we rename `DashboardsChartReport` to something shorter like `BIReport`, `EmbeddedReport` or `EmbeddedChart`?
"DashboardsChartReport" feels like three times the same thing 😄